File: TypeScript/src/accounts/accountdata.ts
/**
 * Refactor Exercise 2: Account
 *
 * @author Zishen Wen (F22), Deyuan Chen (S22)
 */
class AccountData {

	accountName: string;
	accountNumber: number;

	constructor(accountName: string, accountNumber: number) {
		this.accountName = accountName;
		this.accountNumber = accountNumber;
	}
}

export { AccountData }

File: Java/src/main/java/account/AccountData.java
package account;

/**
 * Refactor Exercise 2: Account
 *
 * @author Zishen Wen (F22), Deyuan Chen (S22)
 */
public record AccountData(String accountHolder, int accountNumber) {

    public int getAccountNumber() {
        return this.accountNumber;
    }

    public String getAccountHolder() {
        return this.accountHolder;
    }
}